index.html

```diff
@@ -291,15 +291,17 @@ <h3>
         </p>
       </section>
     </section>
-    <section>
+    <section data-cite="MEDIAQUERIES-5 manifest-incubations">
       <h2>
-        Addition of `window-controls-overlay` to the manifest
+        Addition of <code><dfn data-export="">window-controls-overlay</dfn>
+        </code> to the manifest
```

Comment:
I believe we want to say 
`Addition of a new [=display mode=]: <code><dfn data-export="">window-controls-overlay</dfn></code>`

Does that work?

Reply:
Yes, done :)